overlord/snapstate/autorefresh.go

```diff
@@ -149,6 +150,44 @@ func (m *autoRefresh) AtSeed() error {
 	return nil
 }
 
+func (m *autoRefresh) canRefreshRespectingMetered(now, lastRefresh time.Time) (can bool, err error) {
+	can, err = canRefreshOnMeteredConnection(m.state)
```

Comment:
The implementation of this internal function might be move right above, so we can read both next to each other and get the logic. Right now it's too far from it, and nothing else uses it.